### PR TITLE
[docs]  Updated documentation for the 402-ingress-nginx module.

### PR DESCRIPTION
**Pull request by Suselz**
## Description
Updated documentation for the 402-ingress-nginx module.
Changes:
* Cloud providers in which the spec `sourceRanges` works are detailed
* The design style of the "Caution!" blocks has been corrected, highlighting has been added as in the pages of other modules

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: fix 
summary: Updated documentation for the 402-ingress-nginx module. Cloud providers that use the `sourceRanges` specification are described.
impact_level: low
```

### CODE REVIEW

File: modules/402-ingress-nginx/crds/ingress-nginx.yaml [hunk 1]
```diff
@@ -170,7 +173,7 @@ spec:
                       description: |
                         Accepts all the incoming `X-Forwarded-*` headers and passes them to upstreams.
 
-                        **Caution!** Make sure that requests to the Ingress are sent from trusted sources when using this option.
+                        > **Caution!** Make sure that requests to the Ingress are sent from trusted sources when using this option.
```
Comment (z9r5) [inline modules/402-ingress-nginx/crds/ingress-nginx.yaml:176]:
```suggestion
                        > **Caution!** Make sure that requests to the Ingress controller are sent from trusted sources when using this option.
```

File: modules/402-ingress-nginx/crds/ingress-nginx.yaml [hunk 2]
```diff
@@ -752,8 +764,8 @@ spec:
                       description: |
                         Annotations that will be passed to service with type load balancer to configure it.
 
-                        **Caution!** The module does not take into account the specifics of setting annotations in different clouds.
-                        Note that you will need to recreate `IngressNginxController` (or create a new controller and then delete the old one) if annotations to provision a load balancer are only used when creating the service.
+                        > **Caution!** The module does not take into account the specifics of setting annotations in different clouds.
+                        >  Note that you will need to recreate `IngressNginxController` (or create a new controller and then delete the old one) if annotations to provision a load balancer are only used when creating the service.
```
Comment (z9r5) [inline modules/402-ingress-nginx/crds/ingress-nginx.yaml:768]:
```suggestion
                        > Note that you will need to recreate `IngressNginxController` (or create a new controller and then delete the old one) if annotations to provision a load balancer are only used when creating the service.
```

**Review by z9r5 — APPROVED** [review]

**Comment by z9r5** [comment]
@Suselz Please fix the DCO test. [The details...](https://github.com/deckhouse/deckhouse/pull/9012/checks?check_run_id=27470444562).

**Comment by Suselz** [comment]
@z9r5, fixed

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [9045](https://github.com/deckhouse/deckhouse/pull/9045) to the branch [release-1.62](https://github.com/deckhouse/deckhouse/tree/release-1.62) successful!

**Pull request merged**